README.md

```diff
@@ -49,6 +49,8 @@ Versions:
 
 SkroutzGreekStemmer Plugin | ElasticSearch
 ---------------------------|------------------
+0.0.13                     | 2.3.1
+0.0.12                     | 1.7.x
```

Comment:
Is `0.0.12` published? I think that this line should not be included in this branch